src/components/Addbook.js

```diff
@@ -26,7 +26,8 @@ const Addbook = () => {
   };
 
   return (
-    <div>
+    <div className="addCard">
```

Comment:
- [ ] Kindly recheck the styling for this section. I am actually not able to see the add book button which is supposed to be present according to the Zeplin design.

**This is how I am seeing your design:**

![image](https://user-images.githubusercontent.com/86318284/201339746-779e3262-10e1-4448-b099-7cbdbc079a08.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/201339814-25cc59b0-0863-4ce6-b7e8-8d11f0ffb865.png)